### ISSUE TITLE
crash when removing a topic with RethinkDB

### ISSUE THREAD

**Issue by Grabber** [tinode/chat#883]
### API call
```
in: '{"del":{"id":"72489","topic":"grpgcH651hEOBI","what":"topic","hard":true}}' sid='nxlLAUAZlN4' uid='3lyuRU12W2c'
```

### log error
```
hub.topicUnreg failed: rethinkdb: Expected type SEQUENCE but found SINGLE_SELECTION:

{
        "Access":       {
                "Anon": 0,
                "Auth": 47
        },
        "CreatedAt":    {
                "$reql_type$":  "TIME",
                "epoch_time":   1692833372.957,
                "timezone":     "+00:00"
        },
        "DelId":        0,
        "Id":   "grpgcH651hEOBI",
        "Owner":        "3lyuRU12W2c",
        "Public":       {
                "fn":   "aaa",
                "note": "␡"
        },
        "SeqId":        0,
        "State":        0,
        "Tags": [],
        "TouchedAt":    {
                "$reql_type$":  "TIME",
                "epoch_time":   1692833372.957,
                "timezone":     "+00:00"
        },
        "Trusted":      null,
        "UpdatedAt":    {
                "$reql_type$":  "TIME",
                "epoch_time":   1692833372.957,
                "timezone":     "+00:00"
        },
        "UseBt":        false
} in:
r.DB("tinode").Table("fileuploads").GetAll(r.Args(r.DB("tinode").Table("topics").Get("grpgcH651hEOBI").Filter(func(var_84 r.Term) r.Term { return r.Row.HasFields("Attachments") }).ConcatMap(func(var_85 r.Term) r.Term { return var_85.Field("Attachments") }).CoerceTo("array"))).Update(func(var_86 r.Term) r.Term { return {UseCount=r.Row.Field("UseCount").Default(1).Sub(1)} })
```

**Comment by mynameisjoe666** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by or-else** [comment]
Please verify the fix https://github.com/tinode/chat/commit/452e5bb160d620f721a46916f63d6178fdd355f8

**Comment by Grabber** [comment]
> Please verify the fix [452e5bb](https://github.com/tinode/chat/commit/452e5bb160d620f721a46916f63d6178fdd355f8)

Still testing, but it seems to be working now:
```
{"id":"97083","topic":"grpMXPmOTuSUR4","code":200,"text":"ok","ts":"2023-08-24T15:30:00.629Z"}
```

**Comment by Grabber** [comment]
It's ok.
